frontend/css/search-track.less

```diff
@@ -11,24 +11,23 @@
   border: 1px solid rgba(0, 0, 0, 0.15);
   background-color: @white-background;
   max-height: 102px;
```

Comment:
While in here, could you please remove this line?
Turns out limiting the height of this element is pretty bad UX.